managed-ledger/src/test/java/org/apache/bookkeeper/mledger/impl/ManagedCursorTest.java

```diff
@@ -2278,5 +2280,93 @@ public void testReopenMultipleTimes() throws Exception {
         c1 = ledger.openCursor("c1");
     }
 
+    @Test(timeOut = 20000)
```

Comment:
should we add a test-case which stores `individualDeletedPositions` into ledger and reads back as this PR does this functionality?